pkg/graph/resolvers/trb_user_info_dataloader.go

```diff
@@ -0,0 +1,97 @@
+package resolvers
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/graph-gophers/dataloader"
+
+	"github.com/cmsgov/easi-app/pkg/models"
+)
+
+// Loaders wrap your data loaders to inject via middleware
+type Loaders struct {
+	UserInfoLoader *dataloader.Loader
+}
+
+type ctxKey string
+
+const (
+	loadersKey = ctxKey("dataloaders")
+)
+
+// UserInfoFetcher reads Users from a database
+type UserInfoFetcher struct {
+	FetchUserInfos func(context.Context, []string) ([]*models.UserInfo, error)
+}
+
+// BatchUserInfos implements a batch function to populate UserInfo for EUA IDs
+func (u *UserInfoFetcher) BatchUserInfos(
```

Comment:
Could you add a unit test to verify the data loader? I had some success using errgroups to simulate a dataloader call. This is one implementation in [MINT](https://github.com/CMSgov/mint-app/blob/main/pkg/graph/resolvers/plan_basics_test.go#L48)